### PR TITLE
Add Commit Message Validator

### PR DESCRIPTION
**Pull request by potiuk**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -69,6 +72,11 @@ insertIssueLinkInPrDescription:
     docOnlyIssueMatch:
       titleIssueIdRegexp: \[(AIRFLOW-X{4})\]
       descriptionIssueLink: "`Document only change, no JIRA issue`"
+
+# Verifies if all commit subjects match the regexp specified
+verifyCommitMessages:
+  # Regular expression that should be matched by subject of each commit in the PR
+  commitMessageRegexp: ^.*\[AIRFLOW-[0-9]{4}\].*$|^.*\[AIRFLOW-[X]{4}\].*$
```
Comment (zhongjiajie) [inline README.md]:
```suggestion
  commitMessageRegexp: ^.*\[AIRFLOW-[0-9]{4}\].*$|^.*\[AIRFLOW-X{4}\].*$
```
Some nit here.

Comment (zhongjiajie) [reply README.md]: BTW, should we fix the `AIRFLOW-[0-9]{4}` OR `ARIFLOW-XXXX` in the beginning?
```suggestion
  commitMessageRegexp: ^\[AIRFLOW-[0-9]{4}\].*$|^\[AIRFLOW-X{4}\].*$
```

Comment (zhongjiajie) [reply README.md]: And maybe someone will create PR for an old Jira ticket, so we should use `{1,4}` 
```suggestion
  commitMessageRegexp: ^\[AIRFLOW-\d{1,4}\].*$|^\[AIRFLOW-X{4}\].*$
```

Comment (potiuk) [reply README.md]: I updated a few things now (pointed out by @nuclearpinguin and @mik-laj)
1) the squashed commit can come either from PR title (if you have multiple commits) or from commit (if there is a single commit). So I added a configuration option to check exactly those.

2) @zhongjiajie -> I think we should not worry about old JIRAs. We can always merge those even if we see that commit validation fails (committers can do it). I think there is a value in someone not making an accidental copy&paste without last digit.

**Comment by potiuk** [comment]
And nicer description/screenshots :). Ready to go @kaxil !

**Comment by kaxil** [comment]
Good work @potiuk 🎉

**Pull request merged**